# gets the percentage of relative humidity from the humidity sensor
---
float64 humidity